src/cljs/nr/gameboard/board.cljs

```diff
@@ -658,107 +658,109 @@
   "Returns true if the installed card should be drawn face down."
   [{:keys [side type facedown rezzed host] :as card}]
   (if (= side "Corp")
-    (and (not= type "Operation")
+    (and (and (not= type "Operation")
```

Comment:
nested `and` can be flattened to a single `and`.

Reply:
whoops, I had originally put this as an `or` before realizing that was wrong and just flipped it to an `and`.